utils/cliutils/commandsflags.go

```diff
@@ -1403,6 +1412,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Status,
 		Usage: "[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",
 	},
+	InstallPluginVersion: cli.StringFlag{
+		Name:  version,
+		Usage: "[Optional] The plugin version to download and install.` `",
+	},
+	InstallPluginSrcDir: cli.StringFlag{
+		Name:  InstallPluginSrcDir,
+		Usage: "[Optional] The local directory that contains the plugin files to install.` `",
```

Comment:
Let's add "Use this option to ..."
We should also explain what is the content of this directory.

Reply:
done